src/unix/uiactionx11.cpp

```diff
@@ -127,6 +228,10 @@ class wxUIActionSimulatorPlainX11Impl : public wxUIActionSimulatorX11Impl
     explicit wxUIActionSimulatorPlainX11Impl(wxX11Display& display)
         : wxUIActionSimulatorX11Impl(display)
     {
+        wxYield();
+        wxMilliSleep(50);
```

Comment:
It would be nice to have a symbolic constant for this one too, maybe defined together with the delays used by `wxXSync`, so that they're all in the same place.